.eslintrc.json

```diff
@@ -1,9 +1,10 @@
 {
   "root": true,
   "extends": ["./node_modules/sanctuary-style/eslint-es6.json"],
+  "parserOptions": {"ecmaVersion": 2020, "sourceType": "module"},
   "env": {"node": true},
-  "globals": {
-    "Promise": "readonly"
+  "rules": {
+    "comma-dangle": ["error", {"arrays": "always-multiline", "objects": "always-multiline", "imports": "always-multiline", "exports": "always-multiline", "functions": "never"}]
```

Comment:
This is necessary while we await a sanctuary-scripts release that includes sanctuary-js/sanctuary-style#35.